rootfs/scheduler/__init__.py

```diff
@@ -942,6 +942,8 @@ def _create_rc(self, namespace, name, image, command, **kwargs):  # noqa
         # add in healtchecks
```

Comment:
I know this wasn't introduced in this PR, but let's just fix this spelling mistake. Since we _are_ working on healthchecks here, I think it's allowable.